packages/ui/src/components/va-input/VaInput.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <VaInputWrapper
     v-bind="fieldListeners"
+    :model-value="computedValue"
```

Comment:
```suggestion
```

Reply:
If I won't pass value to the VaInputWrapper I can't get the value `length`, because is value is `undefined` or something.